templates/appliance/governance.html

```diff
@@ -0,0 +1,133 @@
+      <h3 id="experimental" class="p-heading--four">Experimental</h3>
+      <p>This grade can indicate a few things:
+      <ul class="p-link">
+        <li class="p-link__item is-ticked">The snaps included in the image are not all considered to be properly maintained</li>
+        <li class="p-link__item is-ticked">The appliance image itself is undergoing testing by Canonical or by the maintainers</li>
+        <li class="p-link__item is-ticked">The snaps that make up the appliance image are being pulled from the ‘candidate’ or ‘edge’ channels</li>
+      </ul>
+      <p>In the docs section, you will find access to the image builder. A tool where anyone can build appliance images out of whichever snaps they please to test on their own. Any images this way are not guaranteed to work and are also considered experimental.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-8">
+      <h2>Hardware compatibility</h2>
+      <p>Ubuntu Core is guaranteed to work on any <a href="https://certification.ubuntu.com/" class="p-link--external">Ubuntu certified hardware</a>. This includes Raspberry Pi’s and the Intel NUC. Certification means that these boards exist in Canonical’s certification and testing lab and every time an update, bug fix or security patch is rolled out to the OS it is tested on these boards first. This is to ensure no update breaks a user’s appliance. Before a project can become an official Ubuntu appliance, it must be tested to work on a Raspberry Pi or Intel NUC.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-8">
+      <h2>Decision-making bodies</h2>
+      <p>In order to have your project published as an official Ubuntu appliance and featured in the portfolio, you must get in touch with the right people at Canonical. The easiest way to do this is through the <a href="https://forum.snapcraft.io/categories" class="p-link--external">snapcraft forum</a>, where Canonical engineers can help you package your software as a snap and test your images. It is then a simple case of creating a post in the snapcraft forum or making a topic in the <a href="https://discourse.ubuntu.com/">Ubuntu Discourse</a>. After that, the decision rests with the <a href="https://docs.google.com/document/d/1DpU0AzAJ4gBa_au4gELBUD3vLz8l-FnAwFdERBHlvwc/edit#heading=h.ifg3ehffcmkc" class="p-link--external">Ubuntu Community Council</a>. The primary community (i.e., non-technical) governance body for the Ubuntu project. Members are appointed by Mark Shuttleworth and approved by a vote by the entire Ubuntu membership.</p>
```

Comment:
is discourse external? also linking to the copy doc

Reply:
It's on a completely different URL in this instance

Reply:
We have started to remove external links to the ubuntu discourse as it is integrated via the header. Even though technically this is another domain for the user it shouldn't. And, there is a native navigation path back to the main site.